packages/eslint-config-nimble/typescript.js

```diff
@@ -32,7 +32,16 @@ module.exports = {
         'require-await': 'off',
         '@typescript-eslint/require-await': 'error',
         'no-return-await': 'off',
-        '@typescript-eslint/return-await': 'error'
+        '@typescript-eslint/return-await': 'error',
+
+        // Increasing the value of minProperties (default is 6?) so that eslint doesn't
+        // introduce line breaks where prettier doesn't. If eslint introduces line breaks,
+        // they will be unix-style, which will cause pointless diffs in git.
+        'object-curly-newline': ['error', {
```

Comment:
The docs say this rule (and all other formatting related rules, apparently) was deprecated in eslint 8.53.0 (we're on 8.57.0). The recommendation is to get the same rules from a different package, `@stylistic/eslint-plugin-js`, but I didn't want to go down that road right now. If this stops being supported or we get a bunch of ugly warnings in the future, we can deal with it then.

Reply:
Yea I'd expect that transition should happen in the ni style-guide. Could you create an issue to point out the deprecated rules there?

Reply:
Created https://github.com/ni/javascript-styleguide/issues/148